<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="leftGraphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>21</y>
      <width>281</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="rightGraphicsView">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>281</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="methodComboBox">
    <property name="geometry">
     <rect>
      <x>835</x>
      <y>40</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="windowSizeEdit">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>70</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>170</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="computeButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>170</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="disparityGraphicsView">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>281</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxDisparityEdit">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>100</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="loadLeftButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadRightButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>240</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
